localstack/services/awslambda/invocation/executor_endpoint.py

```diff
@@ -26,10 +26,18 @@ def __init__(self, invocation_id: str, payload: Optional[bytes]):
 
 class ExecutorEndpoint(Server):
     service_endpoint: ServiceEndpoint
-
-    def __init__(self, port: int, service_endpoint: ServiceEndpoint, host: str = "0.0.0.0") -> None:
+    port: Optional[int]
+
+    def __init__(
+        self,
+        port: int,
+        service_endpoint: ServiceEndpoint,
+        host: str = "0.0.0.0",
+        container_address: Optional[int] = None,
```

Comment:
shouldn't this be an Optional[str] ?

Reply:
You are right, i was somehow thinking about a port, and then forgot to change it